sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/directconnectivity/RntbdTransportClient.java

```diff
@@ -58,6 +61,26 @@
     private static final AtomicLong instanceCount = new AtomicLong();
     private static final Logger logger = LoggerFactory.getLogger(RntbdTransportClient.class);
 
+    /**
+     * A key that can be used to store a sequence-specific {@link Hooks#onErrorDropped(Consumer)}
+     * hook in a {@link Context}, as a {@link Consumer Consumer&lt;Throwable&gt;}.
+     */
+    private static final String KEY_ON_ERROR_DROPPED = "reactor.onErrorDropped.local";
```

Comment:
@FabianMeiswinkel Is this a feature of Reactor, if so is it documented somewhere? It's interesting.

Reply:
Yes - it is a Reactor feature - but I found it via source code inspection. This is the commit in which it was added: https://github.com/reactor/reactor-core/commit/b576f7f0eb6dbf285759fda15ee95688a79d41bb

Reply:
The best features are always undocumented! My next question is why didn't you use `Operators.KEY_ON_ERROR_DROPPED` and/or provide a link to how it works? :-)

Reply:
Hooks.KEY_ON_ERROR_DROPPED is package internal. But fair point about comment. Will add it.

Reply:
https://github.com/Azure/azure-sdk-for-java/pull/15365

Reply:
Ah! In the commit you posted they were public, but the actual source now is in `Hooks` and yes internal, I get it now! Thanks for adding the comment :-)